### DISCUSSION TITLE
Remove corpses from vehicles

### DISCUSSION THREAD

**Discussion by Jash-ugan** [wolveix/satisfactory-server#21] [Q&A] [answered]
Hey everyone!

My friends and me are using the server extensively and although we encounter some bugs, it works like a charm. One thing that is really annoying is, that if you get thrown out by the server (which occasionally happens) and you are in a vehicle, the vehicle cannot be removed as there is still someone sitting inside. 

Is there any possibility to remove the corpses from the game?

Cheers

**Comment by wolveix** [comment, marked as answer]
Hi! Thanks for opening this discussion :) 

Unfortunately, this is a bug within the game/current server implementation. The easiest way to resolve this is to upload your save to https://satisfactory-calculator.com/en/interactive-map and remove all of the players with empty inventories. I'm working on implementing the removal of "ghost" players in a new update, hopefully releasing soon!

**Reply by Jash-ugan** [reply]
Hey @wolveix! Thanks for your fast reply. I checked out the satisfactory-calculator. Awesome resource, thanks for that. I also created an account there but I cannot find any place where to upload a save there. Can you give me a hint where exactly I have to go?

**Reply by wolveix** [reply]
@Jash-ugan You're welcome! From the interactive map, just click on the upload save section towards the top-right of the page :)
